lib/world/sequential.rb

```diff
@@ -18,6 +18,12 @@ def initialize()
 
         @population_count = Synthea::Config.sequential.population
 
+        @scaling_factor = @population_count / 6794422.0
```

Comment:
The intent was so that if you want to generate a population smaller than 7M but still with accurate ratios, you can scale the populations of individual cities down by this amount.